ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/action-buttons/action-buttons.html

```diff
@@ -24,33 +24,33 @@
      data-sly-test.ready="${actionButtons.ready}">
 
     <div class="ui basic segment">
-        <h4 class="ui header" data-sly-test="${!properties['hideLabel']}">${properties['label']}</h4>
+        <h4 class="ui header" data-sly-test="${!properties['hideLabel']}">${properties['label'] || 'Actions' @ i18n}</h4>
 		<div class="actions cmp-action-buttons">
             <button class="ui primary button cmp-action-buttons__button"
                     data-asset-share-id="download-asset"
                     data-asset-share-asset="${asset.path}"
                     data-asset-share-license="${config.licenseEnabled ? asset.properties['license'] : ''}"
                     data-sly-test="${properties['downloadLabel'] && config.downloadEnabled}">
-                ${properties['downloadLabel']}
+                ${properties['downloadLabel'] || 'Download' @ i18n}
             </button>
             <button class="ui button cmp-action-buttons__button"
                     data-asset-share-id="share-asset"
                     data-asset-share-asset="${asset.path}"
                     data-sly-test="${properties['shareLabel'] && config.shareEnabled}">
-                ${properties['shareLabel']}
+                ${properties['shareLabel'] || 'Share' @ i18n}
             </button>
             <button class="ui button cmp-action-buttons__button"
                     data-asset-share-id="add-to-cart"
                     data-asset-share-asset="${asset.path}"
                     data-asset-share-license="${config.licenseEnabled ? asset.properties['license'] : ''}"
                     data-sly-test.cartEnabled="${properties['addToCartLabel'] && properties['removeFromCartLabel'] && config.cartEnabled}">
-                ${properties['addToCartLabel']}
+                ${properties['addToCartLabel'] || 'Add to Cart' @ i18n}
```

Comment:
@gquintillan Looking at https://github.com/adobe/asset-share-commons/search?q=%22Add%20to%20Cart%22  it seems like we have the label `Add to Cart` already in many places, wouldn't just `${properties['addToCartLabel'] @ i18n}` work fine here as well?

Reply:
@gquintillan what are your thoughts on @ghenzler's suggestion? when looking at the PR it does feel like a lot of duplication between the template and the HTML file. Do you see any problems w/ @ghenzler 's approach that would break for you?

Reply:
Hi, yes I could remove all the `|| 'X'` that I added, I just took the downloads component as an example:
https://github.com/adobe/asset-share-commons/blob/261238d49c5e16b2e40f29f347c80801cd3c330f/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/downloads/downloads.html#L88

I wanted to follow the same approach because I didn't want to risk, I thought that maybe in previous versions the _cq_template node was empty and I wanted to avoid problems for instance if someone installs the new version and doesn't have the default values.

Reply:
But I agree that it is a lot of duplication (which I don't like), let me try to remove all the `|| 'X'` that I added, do some testing and see if something happens.

Reply:
Thanks @gquintillan this looks nice!

Reply:
cool, thanks!